### ISSUE TITLE
`sys_bin` option is set to a GHA runner dir in TinyTeX bundle 

### ISSUE THREAD

**Issue by cderv** [rstudio/tinytex-releases#26]
@yihui 

See this when installing on Arch Linux 
````bash
# /root/.TinyTeX/bin/x86_64-linux/tlmgr option sys_bin
Destination for symlinks for binaries (sys_bin): /home/runner/bin
````

The installation was done with `quarto install tool tinytex` 

This means that when doing `tlmgr path add` with the monthly bundle version, then `/home/runner` is created. I don't think that is right. 

I am seeing that we do a post install configuration on TinyTeX: 
https://github.com/cderv/tinytex/blob/main/R/install.R#L461-L480

Is this to solve such issues ? Or was it for something else ? 

is the fact that `tlmgr option sys_bin` is set to a value from the GH runner a limitation of our bundles ? 

Maybe it is known and already documented but somehow I was suprised.

**Comment by cderv** [comment]
For the context, we get `/home/runner/bin` because we do configure to use `$HOME/bin` and it gets resolved as absolute. 
The config is still there in the bundle we release. Can we clean this maybe ? Do we need to reset it somehow ? 🤔

**Comment by yihui** [comment]
Yes, the installation script should run `tlmgr option sys_bin`. I did it in both R functions (as you mentioned above) and the shell script, e.g., https://github.com/rstudio/tinytex/blob/5f0cb70e02877f46d806447a362c58ff7dc2748f/tools/install-bin-unix.sh#L63-L68 It sounds like quarto didn't do it?

**Comment by cderv** [comment]
> It sounds like quarto didn't do it?

Quarto does not do it anymore since a choice as been made to not add to PATH by default. It will do it when `quarto install tool tinytex --update-path` is run only.  I add a quick discussion about that with @dragonstyle just earlier today. I was wondering if we should do it anyway or not...

**Comment by yihui** [comment]
I think you should, since it's harmless when you don't add to `PATH`, and is definitely necessary when you do.
